starter-code/scripts/models/article.js

```diff
@@ -46,16 +46,23 @@ Article.fetchAll = function() {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    var data = jQuery.parseJSON(localStorage.blogArticles);
+    Article.loadAll(data);
+    Article.toHtml();
   } else {
     /* Without our localStorage in memory, we need to:
     1. Retrieve our JSON file with $.getJSON
       1.a Load our json data
       1.b Store that data in localStorage so that we can skip the server call next time,
       1.c And then render the index page.*/
+    $.getJSON('data/blogArticles.json').done(function(data) {
```

Comment:
Interesting use of the .done() jQuery method! Can you think of another way to incorporate a callback function as part of your getJSON method?